mmv1/products/firebase/terraform.yaml

```diff
@@ -57,6 +57,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           - project
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/self_link_as_name.erb
+  AppleApp: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ['{{project}} {{name}}']
+    autogen_async: true
+    skip_delete: true #currently only able to delete a appleapp through the Firebase Admin console
```

Comment:
We support Delete on AppleApp, so lets not do this.

we should not pull the new resource in crippled.  Slower updated to WebApp are not as big an issue as it already has that gap.